src/test/java/utils/PropertiesReader.java

```diff
@@ -0,0 +1,49 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesReader {
+    static private PropertiesReader _instance = null;
+    //Variables for the properties
+    public String url = null;
+    public String authorization = null;
+
+    Properties properties = new Properties();
+
+    private static final String PROPERTIES_FILE_PATH = "resources.properties";
+
+    String propertyUrl = "rest.projects.url";
+    String authorizationString = "rest.projects.authorization";
+
+    protected PropertiesReader() {
+        InputStream input = null;
+        try {
+            input = new FileInputStream(PROPERTIES_FILE_PATH);
+            properties.load(input);
+            //List of preloaded properties
+            url = properties.getProperty(propertyUrl);
+            authorization = properties.getProperty(authorizationString);
```

Comment:
I think you need to apply single responsability to this constructor
Move the assignations of these variables in methods